drishyauk/token-assessment
# feat:added dashboard for crypto tracker

### Steps to test:

- Clone the branch
- run `npm i` inside the cloned repo
- run `npm start`
- you will be able to see the search bar, currency selector and top 50 cyrpto

<img width="1437" alt="Screenshot 2024-11-29 at 12 34 31 PM" src="https://github.com/user-attachments/assets/24f3ecd7-7fcc-4a3a-ab3d-b68d9b4a4e7b">

- Enter any crpto name like BNB or Crpto, click on search, observe that it gets added to recent searches.
- Do the same for other crypto

<img width="1288" alt="Screenshot 2024-11-29 at 12 35 59 PM" src="https://github.com/user-attachments/assets/30faa565-6846-4117-bd52-8aeb77072155">
